README.md

```diff
@@ -26,92 +26,32 @@ The resulting genome files and tool indices will be located in the directory spe
 
 The three important files are:
```

Comment:
```suggestion
The two important files are:
```

Reply:
Will follow up with this fix - thanks!

Reply:
Just merged that fix directly to main. Thanks again